<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshAnalyzer</class>
 <widget class="QDialog" name="MeshAnalyzer">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mesh analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/Images.qrc">
    <normaloff>:/img/app-icon.png</normaloff>:/img/app-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelPoints">
     <property name="text">
      <string>&lt;b&gt;Points:&lt;/b&gt; 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTriangles">
     <property name="text">
      <string>&lt;b&gt;Triangles:&lt;/b&gt; 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/Images.qrc"/>
 </resources>
 <connections/>
</ui>
